keyboards/launchpad/config.h

```diff
@@ -21,11 +21,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0x1337
-#define PRODUCT_ID      0x6007
-#define MANUFACTURER    Maple Computing
-#define PRODUCT         Launch Pad
-#define DESCRIPTION     An all-in-on macropad
+#define VENDOR_ID 0x1337
```

Comment:
Not sure if the changes to this block are intentional - maybe this whole file should be reverted.

Reply:
reverted the whole file